packages/react-components/src/form.ts

```diff
@@ -40,6 +40,12 @@ export const styles = {
   ':focus': {
     borderColor: fern.rgb,
   },
+  ' ~ div svg': {
```

Comment:
This seems a bit spooky, how can `form.ts` know about the `div` structure of the various form elements